index.js

```diff
@@ -1,31 +1,48 @@
 /* jshint node:true */
 'use strict';
 
-var getHoodieServer = require('hoodie');
+var Hapi = require('hapi');
+var hoodie = require('hoodie');
 var url = require('url');
 var path = require('path');
 var proxy = require('http-proxy-middleware');
 
 var mergeTrees = require('broccoli-merge-trees');
 var Funnel = require('broccoli-funnel');
 
-var hoodie_options = {
-  port: 4201
+var hapiOptions = {
+  server: {},
+  connection: {
+    port: 4201
+  }
+};
+var hoodieOptions = {
+  paths: {
+    public: 'dist'
+  }
 };
 
 function hoodieMiddleware(config) {
-  config.app.use('/hoodie', proxy({target: 'http://localhost:' + hoodie_options.port}));
+  config.app.use('/hoodie', proxy({target: 'http://localhost:' + hapiOptions.connection.port}));
+
+  var server = new Hapi.Server(hapiOptions.server);
+  server.connection(hapiOptions.connection);
 
-  getHoodieServer(hoodie_options, function (error, server, config) {
+  server.register({
+    register: hoodie,
+    options: hoodieOptions
+  }, function (error) {
     if (error) {
-      var stack = new Error().stack.split('\n').slice(2).join('\n');
-      return console.log('app', 'Failed to initialise:\n' + stack, error);
+      throw error;
     }
 
     console.log('app', 'Starting');
+    server.start(function (error) {
+      if (error) {
+        throw error;
+      }
 
-    server.start(function () {
-      console.log('Your Hoodie server has started on ' + url.format(config.connection));
+      console.log('Your Hoodie server has started on ' + url.format(server.info.uri));
```

Comment:
I don't use ember, but is this relevant? http://emberjs.com/api/classes/Ember.Logger.html

I was curious haha